perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/items/PerunFormItemEditable.java

```diff
@@ -31,6 +31,8 @@ public abstract class PerunFormItemEditable extends PerunFormItem {
 	private HelpBlock help;
 	public PerunRegistrarTranslation translation = GWT.create(PerunRegistrarTranslation.class);
 
+	private static final int DEFAULT_MINIMUM_PASSWORD_LENGTH = 8;
```

Comment:
i think it might be better to reuse the constant from Password.MIN_LENGTH, that way a single change of the value would propagate to all the necessary places...